src/app/work-item/work-item-board/work-item-board.component.ts

```diff
@@ -353,8 +354,35 @@ export class WorkItemBoardComponent implements OnInit, OnDestroy {
   }
 
   onMoveToBacklog(event: any): void {
-    alert('Not Implemented yet');
     event.stopPropagation();
+    //set this work item's iteration to None
+    //send a patch request
+    this.workItem.relationships.iteration = {}
```

Comment:
Pasting Mattermost chat:
> 
nimisha 
5:05 PM @michaelkleinhenz (pandora's box) on the kebab menu for Move to Backlog - what do I set the iteration as {} or root 

michaelkleinhenz 
5:07 PM @nimisha {} should work to set the iteration to root.
5:08 PM @nimisha so "iteration: {}", not "iteration: {data: {}}"
5:08 PM @nimisha if it is not working, it is a bug in the core.

Core task to handle empty iteration - https://github.com/almighty/almighty-core/issues/1185